src/test/groovy/generated/AutoExternalizeGeneratedTest.groovy

```diff
@@ -0,0 +1,44 @@
+package groovy.generated
+
+import groovy.transform.CompileStatic
+import org.junit.Ignore
+import org.junit.Test
+
+/**
+ * @author Dmitry Vyazelenko
+ * @author Andres Almiray
+ */
+@CompileStatic
+class AutoExternalizeGeneratedTest extends AbstractGeneratedAstTestCase {
+    final Class<?> implicitAutoExternalize = new GroovyClassLoader().parseClass('''@groovy.transform.AutoExternalize
+       |class ClassUnderTest {
+       |}'''.stripMargin())
```

Comment:
Nothing wrong with `stripMargin` but for consistency with other tests (and to reduce unnecessary noise) I'd suggest just getting rid of it and the `|` characters since Groovy ignores whitespace - unless you were wanting to test specific column numbers of error messages or similar where it would make sense to keep.